### PR TITLE
feat(vercel): Added completion for vercel's alias, link, billing

### PR DESCRIPTION
**Pull request by alexnaiman**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Added completion for for vercel's alias, link, billing
**What is the current behavior? (You can also link to an open issue here)**
Missing completion specs for alias, link, billing. See #694 
**What is the new behavior (if this is a feature change)?**
Completion for vercel's alias, link, billing
**Additional info:**
none

### CODE REVIEW

File: src/vercel.ts [hunk 1]
```diff
@@ -653,6 +653,73 @@ const completionSpec: Fig.Spec = {
...
+      name: "alias",
+      description:
+        "Apply custom domains based on git branches, or other heuristics",
+      subcommands: [
+        {
+          name: "set",
+          description: "Assign a custom domain to a deployment",
+          args: [
+            {
+              name: "deployment url",
```
Comment (QuiiBz) [inline src/vercel.ts:666]: The name of the argument should be one word if possible, and then you can add a `description` to explain a bit more about it. Can you change this for each argument underneath?

As example:
```suggestion
              name: "url",
              description: "The URL of the deployment to assign to a domain",
```

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks a lot for this PR! There's only one issue that needs to be fixed at few places

File: src/vercel.ts (hunk 1 above)
Comment (mschrage) [reply src/vercel.ts:666]: I wonder if we can set up a linter for this? cc @fedeci

Comment (QuiiBz) [reply src/vercel.ts:666]: That would be a good idea!

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks a lot for this contribution!

**Pull request merged**